tests/themes-project/themes/blog_theme/theme.ini

```diff
@@ -0,0 +1,15 @@
+name = "Blog theme"
+license = "MIT"
+licenselink = ""
+description = "And Example blog theme for testing"
+homepage = ""
+tags = ["simple", "minimal", "blog"]
+features = ["blog"]
+lektor_required_version = 3.1
+
+[author]
+    name = ""
+    homepage = ""
+
+[packages]
```

Comment:
Maybe this should be left out if package/plugin support is not in yet?

Reply:
I added a TODO